elyra/pipeline/handlers.py

```diff
@@ -51,8 +56,41 @@ def post(self, *args, **kwargs):
         self.write(json_msg)
         self.flush()
 
-    def __artifact_list_to_str(self, pipeline_array):
-        if not pipeline_array:
-            return "None"
+
+class PipelineSchedulerHandler(HttpErrorMixin, APIHandler):
+    """Handler to expose method calls to execute pipelines as batch jobs"""
+
+    @web.authenticated
+    @gen.coroutine
+    def get(self):
+        msg_json = dict(title="Operation not supported.")
+        self.write(msg_json)
+        self.flush()
+
+    @web.authenticated
+    @gen.coroutine
+    def post(self, *args, **kwargs):
+        self.log.debug("Pipeline SchedulerHandler now executing post request")
+
+        pipeline_definition = self.get_json_body()
+
+        self.log.debug("JSON payload: %s", pipeline_definition)
+
+        pipeline = PipelineParser.parse(pipeline_definition)
+
+        if pipeline.export:
+            PipelineProcessorManager.export(pipeline)
```

Comment:
We should throw an exception here.  The client is calling the wrong endpoint.

The same goes for `PipelineExportHandler`.  It should confirm that `pipeline.export` is True.

Reply:
that has been removed from here with `rename variables and clean up` pr